src/care/KeyValueSorter.h

```diff
@@ -1135,15 +1135,15 @@ class KeyValueSorter<T, RAJA::seq_exec> {
             std::sort(rawData, rawData + lsize, cmpKeys<T>);
 
             // Reallocate memory
-            if (m_keyValues) {
+            if (m_keyValues != nullptr) {
```

Comment:
The `if (ptr)` pattern should not need to be changed.

Reply:
Reverted